HerPublicWebsite/Views/Questionnaire/NotParticipating.cshtml

```diff
@@ -0,0 +1,111 @@
+﻿@using GovUkDesignSystem
+@using GovUkDesignSystem.GovUkDesignSystemComponents
+@using HerPublicWebsite.BusinessLogic.Models.Enums
+@using Microsoft.AspNetCore.Mvc.TagHelpers
+@using HerPublicWebsite.Controllers
+@using HerPublicWebsite.Models.Enums
+
+@model HerPublicWebsite.Models.Questionnaire.NotParticipatingViewModel
+
+@{
+    ViewBag.Title = "Your local authority is not taking part"; 
+}
+
+@section BeforeMain {
+    <nav>
+        @await Html.GovUkBackLink(new BackLinkViewModel
+        {
+            Text = "Back",
+            Href = Model.BackLink,
+        })
+    </nav>
+}
+
+<div class="govuk-grid-row">
+    <div class="govuk-grid-column-two-thirds">
+        
+        <h1 class="govuk-heading-l">Your Local Authority is not receiving applications via this service.</h1>
+        
+        <h2 class="govuk-heading-m">What to do next</h2>
+
+        @await Html.PartialAsync(Model.LocalAuthorityMessagePartialViewPath)
+
+        <p class="govuk-body">
+            If your local authority does not seem right, <a class="govuk-link" href="https://www.gov.uk/find-local-council">find your correct local authority</a> and contact them directly.
+        </p>
+        
+        @if (Model.Submitted)
+        {
+            @await Html.GovUkNotificationBanner(
+                       new NotificationBannerViewModel
+                       {
+                           Type = "success",
+                           TitleText = "Success",
+                           Text = "Your contact details have been submitted.",
+                           Role = "alert"
+                       }
+                       )
```

Comment:
I realise that this is copied from `Pending.cshtml`, but the formatting is incorrect. There are a number of similar issues throughout the file.

In Rider, please run _Code_ > _Reformat and Cleanup_ on just this file, with the "Full Cleanup" profile. That will fix everything.

(Alternatively, you can use CTRL+SHIFT+A and type to find the "Silent Reformat and Cleanup" action.)